### PR TITLE
skaffold 1.6.0

### PR DESCRIPTION
**Pull request by balopat**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/skaffold.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@ class Skaffold < Formula
   end
 
   # relates to https://github.com/GoogleContainerTools/skaffold/pull/3775
-  # should be updated to use go after next release (> v1.5.0)
```
Comment (chenrui333) [inline Formula/skaffold.rb]: Just curious why can it not move to go@1.14?

Comment (balopat) [reply Formula/skaffold.rb]: we should! I'm going to update this - also we had to retag due to a bug in the release process, so I'll update the commit too

**Comment by chenrui333** [comment]
That looks much better, thanks @balopat!!

**Pull request closed without merging**